src/libraries/Directory.Build.props

```diff
@@ -103,6 +103,14 @@
     <IsPackable Condition="$(MSBuildProjectName.StartsWith('Microsoft.Extensions.')) and '$(IsSourceProject)' == 'true'">true</IsPackable>
   </PropertyGroup>
 
+  <Import Sdk="Microsoft.DotNet.PackageValidation" Project="Sdk.props" Condition="'$(IsPackable)' == 'true' and '$(IsSourceProject)' == 'true'"/>
+
+  <PropertyGroup Condition="'$(IsPackable)' == 'true'">
+    <PackageValidationBaselineVersion>5.0.0</PackageValidationBaselineVersion>
+    <PackageValidationBaselineName>$(MSBuildProjectName)</PackageValidationBaselineName>
```

Comment:
Is this the best property to use?  Shouldn't we look at what Pack task uses for [PackageId](https://github.com/NuGet/NuGet.Client/blob/afa624edca69695b1b6be112b62a9157c36922d5/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L225)?

Reply:
You don't nee to set this at all. The validation SDK already defaults to PackageId.

Reply:
Good point (y)

Reply:
> You don't nee to set this at all. The validation SDK already defaults to PackageId.

yes thats correct but currently the upfront restore is not working as expected. still debugging, the packages are not being restored and nuget.g.props files are not getting generated, it works fine while building a single project

Reply:
How does setting the baseline name help with that?

Reply:
> How does setting the baseline name help with that?

it doesnt help the restore, it helped me test everything when the projects are restored in cache :P

Reply:
Do you need help with the upfront restore issue? I can take a look this week.

> it doesnt help the restore, it helped me test everything when the projects are restored in cache :P

Please remove that line then :)